kits/arms/ex_mobile_device_control.m

```diff
@@ -195,45 +221,49 @@
         cmd.position = nan(1,numModules);
         cmd.velocity = nan(1,numModules);
 
-        % Update base orientation
-        q = [ fbk.orientationW(1) ...
-              fbk.orientationX(1) ...
-              fbk.orientationY(1) ...
-              fbk.orientationZ(1) ];
-        R_base = R_base_init' * HebiUtils.quat2rotMat( q );
-
 
         %%%%%%%%%%%%%%%%%%%%%%%%%%%%
         % Read/Map Joystick Inputs %
         %%%%%%%%%%%%%%%%%%%%%%%%%%%%
         
         % Check for restart command
-        if fbkPhoneIO.b1
+        if fbkPhoneIO.(resetPoseButton)
             break;
         end
         
+        % Check for quit command
+        if fbkPhoneIO.(quitDemoButton)
+            abortFlag = true;
+            break;
+        end
+        
+        if armParams.hasGripper
+            gripperCmd.effort = gripForceScale * ...
+                                    latestPhoneIO.(gripForceSlider) + ...
+                                    gripForceShift;
+            gripperGroup.send(gripperCmd);
+        end
+        
         % Parameter to limit XYZ Translation of the arm if a slider is
-        % pulled down.
-        phoneControlScale = (fbkPhoneIO.a3 + 1) / 2 ;
-
-        % Pose Informatin for arm Control
-        % CURRENTLY HACKED ON ALTERNATE FEEDBACK CHANNELS UNTIL API UPDATE.
-        xyzPhoneNew = [ fbkPhone.position ...
-                        fbkPhone.velocity ...
-                        fbkPhone.effort ];
-                    
+        % pulled down.  Pulling all the way down resets translation.
+        phoneControlScale = (fbkPhoneIO.(translationScaleSlider) + 1) / 2 ;
```

Comment:
the default slider value of zero makes this really awkward. I'd vote for just using values from zero to one side. It's mostly used as a binary switch anyways.

(or we make the app sliders default to `-1` like Linux joysticks (I think?))